### DISCUSSION TITLE
Documentation?

### DISCUSSION THREAD

**Discussion by bycolour2** [themesberg/flowbite#279] [General]
Do you mind add any documentation on react component library?

**Comment by zoltanszogyenyi** [comment]
Hello @bycolour2,

The React community is already working on this based on the main docs layout.

Currently, you can check out this: https://flowbite-react.com/

**Reply by bycolour2** [reply]
Yea, im already using this, but lack of info about props, method and other stuff. Little bit hard for junior developers to use it :)
But thanks for reply, really nice of you

**Reply by kdubb1337** [reply]
I agree. I'm especially stumped about how theming works and how to import components without pulling the whole lib into your first load. The docs don't cover it and you have to look for deep cuts on discussions and sift through code commits.

**Reply by Jack-Koriel** [reply]
I have a similar issue about these docs for react. I am unable to find online something pretty basic like "how to actually import a component"
Most likely I will move from Flowbite to a better documented library.

**Reply by zacBkh** [reply]
Same issue here, the documentation needs to be reworked and provide concrete examples on how to use the components (including imports statements etc)
Thanks a lot for the amazing package!
